### PR TITLE
Add services search with Elasticsearch

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

**Review by evrusha — APPROVED** [review]

File: app/controllers/search_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class SearchController < ApplicationController
```
Comment (fooooxmr) [inline app/controllers/search_controller.rb]: Campaign::SearchController

Comment (fooooxmr) [reply app/controllers/search_controller.rb]: campaign/3/search

File: app/models/service.rb [hunk 2]
```diff
@@ -9,4 +14,6 @@ class Service < ApplicationRecord
   validates :duration, presence: true, numericality: { greater_than: 0, only_integer: true }
   validates :price, presence: true, numericality: { greater_than: 0 }
 
+  Service.__elasticsearch__.create_index!
```
Comment (fooooxmr) [inline app/models/service.rb]: Should be done only once.

File: config/routes.rb [hunk 3]
```diff
@@ -16,7 +16,11 @@
     }, skip: :omniauth_callbacks
 
     resources :companies, only: %i[show edit update] do
-      resources :services
+      resources :services do
+        collection do
+          get 'search', to: 'search#search'
```
Comment (fooooxmr) [inline config/routes.rb]: you can move search to services controller.

File: app/models/service.rb (hunk 2 above)
Comment (ritamma) [reply app/models/service.rb]: Done.

File: config/routes.rb (hunk 3 above)
Comment (ritamma) [reply config/routes.rb]: Done

**Pull request merged**